ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -361,6 +367,13 @@ func (scbuilder *SiteConfigBuilder) getExtraManifest(dataMap map[string]interfac
 		}
 	}
 
+	// merge the pre-defined manifests
```

Comment:
Maybe it's not that important, but for completeness I'd actually prefer this get moved until after the "workload partitions" MC, and the explicitly exclude that one from merging as well.  So basically it groups:
(1) All our provided MCs (some of which should not be merged)
(2) The merge step
(3) All customer-provided MCs